src/serialize/root.zig

```diff
@@ -0,0 +1,133 @@
+const std = @import("std");
+const testing = std.testing;
+const Allocator = std.testing.allocator;
+
+const MAX_FIELD_COUNT: u8 = 255;
+
+pub fn tomlize(allocator: std.mem.Allocator, obj: anytype, writer: anytype) !void {
```

Comment:
Let's name it `serialize`, because we are already anyway in the context of toml. So the usage will be `toml.serialize(...`